client/Lykke.Service.FakeExchange.Client/FakeExchangeClient.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Lykke.HttpClientGenerator;
+
+namespace Lykke.Service.FakeExchange.Client
+{
+    public class FakeExchangeClient : IFakeExchangeClient
+    {
+        //public IControllerApi Controller { get; }
```

Comment:
Commented lines

Reply:
It's from the template, `Lykke.Service.FakeExchange.Client` and `Lykke.Service.FakeExchange.AzureRepositories` projects are staying in the initial state for now